srcpkgs/ncspot/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'ncspot'
+pkgname=ncspot
+version=0.1.4
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config python3"
+makedepends="pulseaudio-devel dbus-devel libressl-devel ncurses-devel"
+depends="pulseaudio dbus libxcb libssl48 ncurses"
```

Comment:
All of these `depends` should be detected automatically as dynamic libraries that this package is linked to, so you don't need to list it here. You might want to have `libxcb-devel` in `makedepends` as well.